app/models/manager_refresh/save_collection/saver/base.rb

```diff
@@ -67,6 +67,10 @@ def save!(inventory_collection, association)
           end
         end
 
+        unless inventory_collection.custom_reconnect_block.nil?
+          inventory_collection.custom_reconnect_block.call(inventory_collection, inventory_objects_index, attributes_index)
+        end
```

Comment:
So for objects that are re-connected by this block, what prevents them from being created by the next section?